internal/recoverypipeline/bundle.go

```
@@ -0,0 +1,353 @@
+// the budget, kept or not, so a stream of torn entries cannot drive unbounded work.
+func expandBundle(zipPath, tempDir string, presentation ErrorPresentation) (
+	artifacts []Artifact, info *BundleInfo, warnings []Warning, err error) {
+
+	sourceID := filepath.Base(zipPath)
+
+	reader, err := zip.OpenReader(zipPath)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("⚠ unable to open bundle `%s`: %s",
+			presentation.path(zipPath), presentation.err(err))
+	}
+	defer reader.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `reader.Close` is not checked

(errcheck)